.github/workflows/close-stale-feature-requests.yml

```
@@ -19,9 +19,10 @@ env:
 
   WARN_MESSAGE: >
     There has been no activity on this feature request for
-    5 months and it is unlikely to be implemented.
-    It will be closed 6 months after the last non-automated comment.
-
+    5 months and to help maintain relevant open issues please
+    add the never-stale label or close this issue if it should be
+    closed. If not the issue will be automatically closed 6
+    months after the last non-automated comment.
```

```suggestion
    5 months. To help maintain relevant open issues please
    add the never-stale label or close this issue if it should be
    closed. If not, the issue will be automatically closed 6
    months after the last non-automated comment.
```

---

I am not certain what the proper english structure would be here, but re-reading this again I am pretty sure it would be better to add a `,` after `issues` than to combine these sentences? Not sure and I prefer not to bike-shed this kind of thing. I see it comes with an approval so what do you think about going with my added comma instead?

---

To be clear, I meant "in addition to your other added comma"

---

I'm not a native speaker, feel free to ignore.

---

No worries! I want to make sure it is a clear as we can, including to as many of our colleagues and contributors around the world. So your input is super valuable to me to make that happen. I just want to understand if the following would also help clarify the language:

```suggestion
    5 months. To help maintain relevant open issues, please
    add the never-stale label or close this issue if it should be
    closed. If not the issue will be automatically closed 6
    months after the last non-automated comment.
```

I *think* this is technically more correct lol, but really I am also not an expert or trying to be pedantic, so whatever is the most clear *and* correct is what I would like to go with.